samples/nrf9160/nrf_cloud_rest_device_message/src/main.c

```diff
@@ -344,6 +349,35 @@ static int setup(void)
 	return 0;
 }
 
+int send_hello_world_msg(void)
+{
+	int err;
+	int64_t time_now;
+	char buf[SAMPLE_MSG_BUF_SIZE];
+
+	/* Get the current timestamp */
+	err = date_time_now(&time_now);
+	if (err) {
+		LOG_ERR("Failed to get timestamp for Hello World message");
+		return err;
+	}
+
+	/* Send off a hello world message! */
+	/* This does not match a predefined schema. But that's not a problem! */
+	err = snprintk(buf, SAMPLE_MSG_BUF_SIZE, SAMPLE_MSG, time_now);
```

Comment:
This prints the full time stamp as an integer, right? Not just the MS?

That's not a problem, I prefer it to be the full timestamp, but I want to verify since I have that preference

Reply:
Yes, it prints the whole thing as an int.